modal_proto/api.proto

```diff
@@ -2875,6 +2875,7 @@ message VolumeMount {
   string volume_id = 1;
   string mount_path = 2;
   bool allow_background_commits = 3;
```

Comment:
Is this deprecated btw?

Reply:
I think so. Not sure if you can force disable it tbh.

Reply:
The backend still respects this setting if you set it to `false`. So we can't remove it just quite yet.